python/py-oci/Portfile

```
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-oci
+version             2.113.0
+revision            0
+
+categories-append   net
+
+license             UPL-1 Apache-2
```

`UPL-1` isn't in the list of licenses that MacPorts understands, unfortunately.  Here's the list: https://trac.macports.org/wiki/PortfileRecipes#license

It may have to be something like:

```suggestion
license             {Permissive Apache-2}
```

---

Done